packages/manager/modules/vps/src/dashboard/terminate/vps-terminate.controller.js

```diff
@@ -1,5 +1,10 @@
 export default class {
-  /* @ngInject */
+  static getDegressivityMonthDetails(degressivityInformation) {
+    const [price] = degressivityInformation.prices;
```

Comment:
Only one price ?

Reply:
All prices of a product have the same pricing mode. So I just need to manipulate one of them, to get degressivity information, no matter which price is used.